apps/studio/components/interfaces/Settings/Database/ConnectionPooling/ConnectionPooling.constants.ts

```diff
@@ -3,27 +3,27 @@
 
 export const POOLING_OPTIMIZATIONS = {
   ci_nano: {
-    maxClientConn: 100,
-    poolSize: 10,
+    maxClientConn: 200,
+    poolSize: 15,
```

Comment:
just a reminder to make sure that the poolSize is not larger than the pg connections limit in nano

Reply:
nano limit is 60 now